### PR TITLE
US020 Delete a role

### PR DESCRIPTION
**Pull request by UserConor**
This pull request contains the UI for the delete a role task.

Criteria:
I want to delete a role
So that I can keep the software information up to date

Acceptance criteria:
Able to delete the role from the job roles list page and the job specification page

### CODE REVIEW

**Review by Joseph-Magee — APPROVED** [review]
looks good

File: model/BandRole.ts
Comment (steveh2707) [inline model/BandRole.ts:1]: this has nothing to do with deleting a role

File: model/JobFamily.ts
Comment (steveh2707) [inline model/JobFamily.ts:1]: this has nothing to do with deleting a role

File: model/RoleFamily.ts
Comment (steveh2707) [inline model/RoleFamily.ts:1]: this has nothing to do with deleting a role

File: service/bandService.ts
Comment (steveh2707) [inline service/bandService.ts:1]: this has nothing to do with deleting a role

File: service/roleService.ts
Comment (steveh2707) [inline service/roleService.ts:1]: this file is a bit of a mess from when you've resolved merge conflicts. The only thing you should be adding is the delete role endpoint.

File: views/job-role-spec-admin.html
Comment (steveh2707) [inline views/job-role-spec-admin.html:1]: is this part of the delete a role feature?

File: views/job-role-spec.html
Comment (steveh2707) [inline views/job-role-spec.html:1]: is this part of the delete a role feature?

File: views/old-roles.html
Comment (steveh2707) [inline views/old-roles.html:1]: is this part of the delete a role feature?

File: views/roles-admin.html
Comment (steveh2707) [inline views/roles-admin.html:1]: is this part of the delete a role feature?

File: views/roles.html
Comment (steveh2707) [inline views/roles.html:1]: is this part of the delete a role feature?

**Pull request merged**